services/nexus/nexus.service.js

```diff
@@ -33,6 +34,15 @@ const resolveApiSchema = Joi.object({
   }).required(),
 }).required()
 
+// https://repository.sonatype.org/nexus-restlet1x-plugin/default/docs/path__artifact_maven_resolve.html
+// https://repository.sonatype.org/nexus-indexer-lucene-plugin/default/docs/path__lucene_search.html
+const queryParamSchema = Joi.object({
+  server: optionalUrl.required(),
+  queryOpt: Joi.string()
+    .regex(/(:(?:q|g|a|v|p|c|cn|sha1|from|count|repositoryId|e|r)=[\w-. ]+)+/i)
```

Comment:
This is a stricter regex than the previous one (`(:.+?)?`), but is based on the query params that the two Sonatype APIs actually accept (details at the links in the above inline comments). 

I can swap this back to the more lenient one if anyone has concerns with this new one